src/app/app.component.scss

```diff
@@ -0,0 +1,134 @@
+$width280: 280px;
```

Comment:
one global scss file where all constants will be defined  mainly color, fonts name and many thing like button